inst/qml/Descriptives.qml

```diff
@@ -176,7 +176,7 @@ Form
 		Group
 		{
 			CheckBox {				name: "descriptivesIntervalPlot";label: qsTr("Interval plots")					}
```

Comment:
for all the `descriptives...Plot` options, you can drop `descriptives`, i.e., 

- `descriptivesIntervalPlot` -> `intervalPlot`
- `descriptivesQQPlot` -> `qqPlot`
etc.